scoring/tests/test_scoring.py

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env python
```

Comment:
I'm not convinced this is a good idea.

`/usr/bin/env python` is not necessarily the interpreter one wants, and one can also run this file with `<interpreter> scoring/tests/test_scoring.py` because explicit is better than implicit. This is also a very much UNIX-specific behaviour.

Reply:
Note that this is in the tests and is a convenience.
The tests should run under all SRComp and libproton supported Pythons, of which `/usr/bin/env python` seems highly likely to be one.